en/docs/references/pricing.md

```diff
@@ -0,0 +1,229 @@
+
+#### Glossary
+
+- <a class="pricing" id="aircraftNotifWebhook">Aircraft Status Notifier</a>:A webhook in Choreo that listens to any events that trigger when the Aircraft maintenance sheet is updated.
+- <a class="pricing" id="AircraftMaintenanceSheet">Aircraft Maintenance Sheet</a>: A Google sheet that maintains the list of aircraft, their information,  the maintenance schedule, and the maintenance status.
+- <a class="pricing" id="flightSchedulingDatabase">Flight Scheduling Database</a>:The database that includes details about  flights, the aircraft assigned to the flight route, and their maintenance state.
+
+<br/>
+
+#### Update maintenance status 
+
+![Update maintenance status](../assets/img/references/webhook-example1.png)
+
+Let’s see how the steps are calculated for the update maintenance process:
+
+- As illustrated above, the webhook registers with the Google sheet at the time the Webhook component is created on Choreo.
+- The supervisor initiates the update process by updating or adding the relevant information on the [Aircraft Maintenance Sheet](#AircraftMaintenanceSheet).
+- The Google Sheet triggers **1 event to the Webhook Listener,[Aircraft Status Notifier](#aircraftNotifWebhook)** on Choreo. This is counted as **1 step**.
+- The Webhook Listener,[Aircraft Status Notifier](#aircraftNotifWebhook) on Choreo then makes **1 database call to the [Flight Scheduling Database](#flightSchedulingDatabase)** to update the flight maintenance status. This is counted as 1 step. 
+
+
+- **Assuming the entire execution time for the update process takes less than 500ms**, the update process thereby results in a **total of 2 steps**.
+
+!!! note
+    The above cost calculation is dependent on the above architecture and design of the Webhooks.
+<br/>
+
+### Example 4: Scheduled Job 
+An airline will revise its ticket prices based on the price of jet fuel. This typically happens once a day. The airline uses a scheduled task on Choreo to run this job and to update the jet fuel price once a day.
+
+#### Scenario 
+ 
+Airlines compute ticket prices typically once per day. The ticket prices depend on a number of factors. The price of oil is one such major factor that affects the pricing.  
+
+The [scheduler](#scheduler) on Choreo runs once per day and fetches the oil prices for the day from the [Oil Price API](#oilPriceAPI). It then computes the jet fuel price applying the price calculation formula. The scheduler then updates the daily jet fuel price in the [Air Fare Database](#airFareDatabase).
+
+Having the jet fuel price, the scheduler is able to now calculate the airfare for each route. The Scheduler retrieves the airline routes from the [Airline Routes API](#airlineRoutesAPI). Upon receiving the airline routes, it iterates over each route, computes the airfare, and updates the AirFare database.
```

Comment:
Shall we change "is able to now" to "is now able to".

Reply:
done